plugins/inputs/snmp/snmp.go

```diff
@@ -308,7 +308,7 @@ func (s *Snmp) Gather(acc telegraf.Accumulator) error {
 			return err
 		} else {
 			for _, line := range strings.Split(string(data), "\n") {
-				oidsRegEx := regexp.MustCompile(`([^\t]*)\t*([^\t]*)`)
+				oidsRegEx := regexp.MustCompile(`([^ \t]*)[ \t]*([^ \t]*)`)
```

Comment:
@titilambert could you remove the regex matching and use `strings.Fields` instead? https://golang.org/pkg/strings/#Fields